utils/directories.cc

```diff
@@ -149,4 +151,108 @@ future<> directories::verify_owner_and_mode(fs::path path, recursive recursive)
     return do_verify_owner_and_mode(std::move(path), recursive, 0);
 }
 
+void directories::set_work_dir(const seastar::sstring& work_dir) {
+    _work_dir = fs::path{work_dir};
+}
+
+void directories::set_commitlog_dir(const seastar::sstring& commitlog_dir) {
+    _commitlog_dir = fs::path{commitlog_dir};
+}
+
+void directories::set_schema_commitlog_dir(const seastar::sstring& schema_commitlog_dir) {
+    _schema_commitlog_dir = fs::path{schema_commitlog_dir};
+}
+
+void directories::set_hints_dir(const seastar::sstring& hints_dir) {
+    _hints_dir = fs::path{hints_dir};
+}
+
+void directories::set_view_hints_dir(const seastar::sstring& view_hints_dir) {
+    _view_hints_dir = fs::path{view_hints_dir};
+}
+
+void directories::set_saved_caches_dir(const seastar::sstring& saved_caches_dir) {
+    _saved_caches_dir = fs::path{saved_caches_dir};
+}
+
+void directories::set_data_file_dirs(const std::vector<seastar::sstring>& data_file_dirs) {
+    std::vector<fs::path> dirs_paths;
+    dirs_paths.reserve(data_file_dirs.size());
+
+    for (const auto& dir_str : data_file_dirs) {
+        dirs_paths.emplace_back(dir_str);
+    }
+
+    _data_file_dirs = std::move(dirs_paths);
+}
+
+static seastar::sstring to_sstring(const fs::path& p) {
```

Comment:
You have `using namespace seastar` up above, so the namespace specifier can be omitted here and all the way down

Reply:
Fixed - thanks :+1: